index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header id="header">
+      <nav id="nav-bar">
+        <img
+          id="header-img"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          width="200"
+          alt=""
+        />
+        <ul>
+          <li><a class="nav-link" href="#Home"> Home</a></li>
+          <li><a class="nav-link" href="#Contact"> contact</a></li>
+          <li><a class="nav-link" href="#Address"> Address</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="Home">
+        <h1>About us</h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis
+          similique assumenda rem soluta suscipit praesentium placeat tempore
+          itaque saepe beatae? Facilis maiores magnam itaque. Libero atque iusto
+          rem laudantium, suscipit facilis temporibus non ipsa qui ad quibusdam
+          sit natus aliquid, velit sunt minus? Necessitatibus, inventore. Porro
+          modi pariatur optio molestiae hic dicta veritatis adipisci iste
+          excepturi amet, magnam repellendus nobis aspernatur similique, dolorum
+          error ex cumque ipsa harum impedit. Amet laboriosam itaque sit nobis
+          possimus, obcaecati labore maiores dolores sunt nostrum libero
+          provident iusto ipsam reiciendis non esse optio perspiciatis deleniti,
+          ab quidem reprehenderit. Perspiciatis dolor ratione impedit voluptates
+          id.
+        </p>
+        <iframe
+          id="video"
+          height="360"
+          src="https://www.youtube.com/embed/kNZnFeULwIw"
+          title="YouTube video player"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </section>
+      <section id="Contact">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur
+          repudiandae sunt praesentium, veritatis animi dolor corrupti eligendi?
+          Delectus, unde tempore?
+        </p>
+        <h2>Request For Newsletters</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            type="email"
+            id="email"
+            placeholder="Email"
+            name="email"
+            required
+          />
+          <input type="submit" id="submit" value="submit" />
```

Comment:
- [ ] Please change the input type `type ="submit"` here is invalid. You can use `type ="button"`.